addons/stock/report/report_stockinventory.xml

```diff
@@ -3,54 +3,50 @@
-                                    <t t-foreach="locations" t-as="location">
-                                        <tr groups="stock.group_stock_multi_locations">
-                                            <td colspan="2"><strong t-esc="location.display_name"/></td>
-                                            <td groups="stock.group_production_lot"></td>
-                                            <td groups="stock.group_tracking_lot"></td>
-                                            <td></td>
-                                        </tr>
-                                        <tr t-foreach="o.line_ids.filtered(lambda line: line.location_id.id == location.id)" t-as="line">
-                                            <td groups="stock.group_stock_multi_locations"></td>
-                                            <td><span t-field="line.product_id"/></td>
-                                            <td groups="stock.group_production_lot"><span t-field="line.prod_lot_id"/></td>
-                                            <td groups="stock.group_tracking_lot"><span t-field="line.package_id"/></td>
-                                            <td class="text-right"><span t-field="line.theoretical_qty"/> <span t-field="line.product_uom_id" groups="uom.group_uom"/></td>
-                                            <td class="text-right"><span t-field="line.product_qty"/> <span t-field="line.product_uom_id" groups="uom.group_uom"/></td>
-                                        </tr>
-                                    </t>
-                                </tbody>
-                            </table>
-                        </div>
-                    </t>
-                 </t>
-             </t>
+                                    <tr t-foreach="docs.filtered(lambda quant: quant.location_id.id == location.id)" t-as="line">
+                                        <td groups="stock.group_stock_multi_locations"></td>
+                                        <td><span t-field="line.product_id"/></td>
+                                        <td groups="stock.group_production_lot"><span t-field="line.lot_id"/></td>
+                                        <td groups="stock.group_tracking_lot"><span t-field="line.package_id"/></td>
+                                        <td class="text-right"><span t-field="line.available_quantity"/> <span t-field="line.product_uom_id" groups="uom.group_uom"/></td>
+                                        <td class="text-right"><span t-field="line.quantity"/> <span t-field="line.product_uom_id" groups="uom.group_uom"/></td>
+                                        <td class="text-right">
+                                            <t t-if="line.inventory_quantity == 0"><span></span></t>
+                                            <t t-else=""><span t-field="line.inventory_quantity"/></t>
+                                            <span t-field="line.product_uom_id" groups="uom.group_uom"/>
```

Comment:
I get this weird looking thing in case of 0 
![image](https://user-images.githubusercontent.com/12071695/112663444-20e45300-8e59-11eb-97d5-740c15ff3458.png)

Reply:
Oh I see. It make sense now